Translations/vi.lproj/FinderExtensionLocalizable.strings

```diff
@@ -1,7 +1,7 @@
-"Open with Keka" = "Open with Keka";
-"Compress here" = "Compress here";
-"Compress to..." = "Compress to...";
-"Extract here" = "Extract here";
-"Extract to..." = "Extract to...";
-"Extract to \"%@\"" = "Extract to \"%@\"";
-"Compress with Keka" = "Compress with Keka";
+"Open with Keka" = "M? v?i Keka";
+"Compress here" = "N?n t?i ??y";
+"Compress to..." = "N?n ??n...";
+"Extract here" = "Gi?i n?n t?i ??y";
+"Extract to..." = "Gi?i n?n ??n...";
+"Extract to \"%@\"" = "Gi?i n?n ??n \"%@\"";
+"Compress with Keka" = "N?n v?i Keka";
```

Comment:
Are those rendered bad?

Reply:
OK. I'll check it back. Thanks.

Reply:
I'd just replaced these broken words via Xcode. Visual Studio Code doesn't work well as saving these.

Reply:
Never used Visual Studio in Mac but probably the encoding for that file was not properly set. Thanks for the fix!